entity-framework/core/miscellaneous/testing/in-memory.md

```diff
@@ -23,9 +23,10 @@ The InMemory provider is useful when you want to test components using something
 EF Core database providers do not have to be relational databases. InMemory is designed to be a general purpose database for testing, and is not designed to mimic a relational database.
 
 Some examples of this include:
-* InMemory will allow you to save data that would violate referential integrity constraints in a relational database.
 
+* InMemory will allow you to save data that would violate referential integrity constraints in a relational database.
 * If you use DefaultValueSql(string) for a property in your model, this is a relational database API and will have no effect when running against InMemory.
+* [Concurrency Tokens](https://github.com/aspnet/EntityFramework.Docs/blob/master/entity-framework/core/modeling/concurrency.md) (`ConcurrencyCheck`, `[Timestamp]`, and `IsRowVersion`) are not supported. No [DbUpdateConcurrencyException](https://docs.microsoft.com/en-us/ef/core/api/microsoft.entityframeworkcore.dbupdateconcurrencyexception) will be thrown if an update is done using an old concurrency token.
```

Comment:
As of 2.1, concurrency tokens and ConcurrencyCheck are supported by the in-memory database. Automatic concurrency--timestamp and rowversion--are still not supported.

Reply:
@ajcvickers 

> Automatic concurrency--timestamp and rowversion--are still not supported.

can u point me to the issue that is tracking that?

Reply:
https://github.com/aspnet/EntityFrameworkCore/issues/10625

Reply:
@ajcvickers fixed

Reply:
@ajcvickers re 10625 https://github.com/aspnet/EntityFrameworkCore/issues/10625#issuecomment-400063921

is there another issue?